README.md

```diff
@@ -63,29 +63,12 @@ If you prefer to install the SDK manually you can use the provided xcframework i
 Bugfender provides a script is able to upload dSYM bundles to [Bugfender](https://bugfender.com).
 
 ## Obtain the script
-If you are uising CocoaPods the script is already distributed with the Pod from version 1.10.4
+The script is distributed if you are using Cocoapods or Swift Package Manager.
```

Comment:
> Swift Package Manager

Is this true now? It wasn't some time ago.

Reply:
Yes, it is part of [the new instructions
](https://github.com/bugfender/BugfenderSDK-iOS/blob/master/xcode-upload-symbols/README.md)

I tested it and it was working